build.gradle

```diff
@@ -1,19 +1,28 @@
+ext {
+    dashjVersion = "0.17.5"
```

Comment:
This was changed to 0.17.6 in 2786ef3de8298ed62a08f62046ad31c915268a64 and the code was moved below as `ext.dashjVersion = "0.17.6"`